lib/db_connection.ex

```diff
@@ -433,6 +433,38 @@ defmodule DBConnection do
   `pid` for connection crashes. So it is recommended to monitor the connected
   `pid` if you want to track all disconnections.
 
+  Here is an example of a `connection_listener` implementation:
+
+    defmodule DBConnectionListener do
+      use GenServer
+
+      def start_link() do
+        GenServer.start_link(__MODULE__, [], name: {:global, "db_connection_listener"})
+      end
```

Comment:
I'd suggest going for a more traditional `start_link/1` here, passing the options to `GenServer.start_link/3`. 

Instead of relying on name for `GenServer.call/3` you can use the pid. It would be nice if this was exported as a function instead of calling `GenServer.call/3` directly.

Reply:
> traditional start_link
Do you think it's better now?

> exported as a function
:white_check_mark: , good call